packages/services/f-braze-adapter/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-braze-adapter",
   "description": "Fozzie Metadata Service",
-  "version": "4.0.0-beta-1",
+  "version": "4.0.0-beta.1",
```

Comment:
Does changing this have any functional difference or is it just personal preference?